src/styles/partials/components/_search.scss

```diff
@@ -0,0 +1,38 @@
+.dg_search-container {
+  position: relative;
+  width: 100%;
+  height: $base-spacing-unit * 10;
+  color: $black;
+
+  .dg_search-btn {
+    display: block;
+    position: absolute;
+    right: $base-spacing-unit * 2;
+    top: $padding-small;
+    font-size: $base-font-size -2;
+    color: $black;
+    cursor: pointer;
+    background: none;
+    border: none;
+
+    i:hover {
+      color: $blue-lightest;
+      transition: all 0.3s ease 0s;
```

Comment:
can you check if this is needed? if you look at _animations.scss i think this is already handled.

Reply:
when this is removed the highlight does not work. So maybe it needs to be added to animations?

Reply:
oh sorry,  the transitions are handled. I'll remove